compiler_gym/envs/llvm/service/ObservationSpaces.h

```diff
@@ -46,6 +46,58 @@ enum class LlvmObservationSpace {
    *     deep reinforcement learning. FCCM.
    */
   AUTOPHASE,
+  /**
+   * The IR2Vec Program Level Flow-Aware embeddings.
+   *
+   * From:
+   *
+   * 	  S. VenkataKeerthy, Rohit Aggarwal, Shalini Jain, Maunendra Sankar Desarkar,
+                                Ramakrishna Upadrasta, and Y. N. Srikant. (2020).
+                                IR2VEC: LLVM IR Based Scalable Program Embeddings.
+                                ACM Trans. Archit. Code Optim. 17, 4, Article 32 (December 2020), 27
+   pages. DOI:https://doi.org/10.1145/3418463
+   *
+   */
+  IR2VEC_FA,
+  /**
+  * The IR2Vec Program Level Symbolic embeddings.
+  *
+  * From:
+  *
+  * 	  S. VenkataKeerthy, Rohit Aggarwal, Shalini Jain, Maunendra Sankar Desarkar,
+                               Ramakrishna Upadrasta, and Y. N. Srikant. (2020).
+                               IR2VEC: LLVM IR Based Scalable Program Embeddings.
+                               ACM Trans. Archit. Code Optim. 17, 4, Article 32 (December 2020), 27
+  pages. DOI:https://doi.org/10.1145/3418463
+  *
+  */
+  IR2VEC_SYM,
+  /**
+* The IR2Vec Function level Flow Aware embeddings.
```

Comment:
Nitpick: indentation of lines 76-86 is off.

Reply:
Fixed!